src/Factory/AdapterFactory.php

```diff
@@ -224,4 +231,10 @@ private function guardFactoryClassImplementation($adapterFactory, $label)
             self::SUPPORT_ISSUE_TRACKER => $issueTracker,
         ];
     }
+
+    private  function getBaseAdapter($name) {
+      $array = explode(':', $name);
+      return $array[0];
+    }
+
```

Comment:
Yes thats true, but i wanted to keep the patch small because currently the URL for the gitlab adapter is stored in ``~/.gush/.gush.yml`` and not the repo.

I think it makes more sense to store it in the repo itself, but it will break BC ...